src/Temporalio/Worker/Interceptors/ScheduleActivityInput.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using Temporalio.Api.Common.V1;
+using Temporalio.Workflows;
+
+namespace Temporalio.Worker.Interceptors
+{
+    /// <summary>
+    /// Input for <see cref="WorkflowOutboundInterceptor.ScheduleActivityAsync" />.
+    /// </summary>
+    /// <param name="Activity">Activity type name.</param>
+    /// <param name="Args">Activity args.</param>
+    /// <param name="Options">Activity options.</param>
+    /// <param name="Headers">Headers.</param>
+    public record ScheduleActivityInput(
```

Comment:
I am using `ExecuteActivity` from a caller POV (see the `Workflows` class). This is just the interceptor, and I have to disambiguate because `ExecuteActivityInput` is already used for activity interceptors (and I didn't want to break this namespace up). This was the subject of the internal Slack thread we had the other day (can link if needed).